### PR TITLE
LPS-150573 Render CommonStylesManager component when the FF is enabled

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/CommonStylesManager.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+			selectedViewportSize === VIEWPORT_SIZES.desktop &&
+			(!isContainerFixed ||
+				(styleName !== 'marginRight' && styleName !== 'marginLeft'))
+		) {
+			filteredStyles[styleName] = styleValue;
+		}
+	});
+
+	return filteredStyles;
+}
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/topper/CommonStylesManager.js:187]: We will need to rethink this in the future if this list of exceptions start to grow :sweat_smile:, but I think it is good enough for now.

**Review by p2kmgcl — COMMENTED** [review]
Probably one of the coolest thing we have added to the page editor :smile:, I love this PR.

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**